sjb/config/test_cases/test_origin_device_manager_plugin_gpu.yml

```diff
@@ -0,0 +1,28 @@
+---
+parent: 'common/test_cases/origin_built_installed_release.yml'
+overrides:
+  junit_analysis: False
+  email: [ "zkosic@redhat.com" ]
+  provision:
+    os: "rhel"
+    stage: "base"
+    provider: "aws"
+    instance_type: "g3.4xlarge"
+extensions:
+  sync:
+    - "zvonkok,openshift-ansible=zvonkok-gpu-origin"
```

Comment:
And just to be clear, we plan to clean this up once the official NVIDIA install on OCP is finalized.